lib/bundler/settings.rb

```diff
@@ -256,5 +280,11 @@ def normalize_uri(uri)
       uri
     end
 
+    def print_remembered_flags_deprecation_warning
+      Bundler.ui.warn "Starting in Bundler 2.0, flags passed into commands will no longer be remember past that command." \
+        " Instead please set flags you want remembered between commands using `bundle config <setting name> <setting value>`" \
+        " To ignore this warning use `bundle config ignore_warnings.remembered_flags true`"
+      set_current "ignore_warnings.remembered_flags", true
```

Comment:
which change? The `set_current`? so that it doesn't show up every time `settings[]=` is called.